build.py

```diff
@@ -1,40 +1,521 @@
 #!/usr/bin/python3
+
```

Comment:
I would love to see most of this file go away. We should just use YAML manifests and `oc apply`, rather than trying to reimplement the behavior of `oc apply` ourselves.

Reply:
But if we need to change anything in the YAML or the JSON files, then we will need to change multiple values correlated over multiple files.  I prefer not to do that on the fly.

Reply:
I think you may not understand the features available to us with things like the `kustomize` support in `oc apply`. Depending on what values you're looking at it, this may not be an issue. And w/r/t screwing up YAML files, you're still maintaining the same data in `build.py`. I don't see a big difference either way.

In any case, we can punt this to a subsequent pull request.

Reply:
I started with yaml files.  I know, they are the preferred way that RedHat does things.  I know you like them, but, and I honestly gave them a try.